projects/cognito-service/src/lib/cognito.service.ts

```diff
@@ -717,6 +722,11 @@ export class CognitoService
     }));
   }
 
+  public logout() : void
```

Comment:
Remove this (`signIn`, `signOut` & `signUp` convention)

Reply:
Done! 😄